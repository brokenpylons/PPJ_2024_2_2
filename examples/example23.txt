var printer = 67;
var for2 = 2;
print(printer * for2);
