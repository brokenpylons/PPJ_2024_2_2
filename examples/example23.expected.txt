real("1.1") 
